huaweicloud/services/dds/resource_huaweicloud_dds_audit_log_policy.go

```diff
@@ -0,0 +1,303 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product DDS
+// ---------------------------------------------------------------
+
+package dds
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceDdsAuditLogPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDdsAuditLogPolicyCreate,
+		UpdateContext: resourceDdsAuditLogPolicyUpdate,
+		ReadContext:   resourceDdsAuditLogPolicyRead,
+		DeleteContext: resourceDdsAuditLogPolicyDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the DDS instance.`,
+			},
+			"keep_days": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				Description:  `Specifies the number of days for storing audit logs.`,
+				ValidateFunc: validation.IntBetween(7, 732),
```

Comment:
remove the validation

Reply:
it can not be removed，because if it is set to 0, then the policy will be closed, and the value range is 7 to 732, so it is necessary to add the validation